atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraReloadableKvsConfig.java

```diff
@@ -256,8 +258,9 @@ public boolean usingSsl() {
 
     @Override
     public void check() {
-        //         The config instance passed here gets checked at the time its of construction
-        //         (default implementation in the parent class). Hence, it is okay to do no operation here.
+        Preconditions.checkState(
```

Comment:
This method is never actually called. It is called by `ImmutableCassandraKeyValueServiceConfig` and we have to override it here, but we don't call it when creating a `CassandraReloadableKvsConfig` because it's not an immutable class.